requirements.in

```
@@ -96,5 +96,5 @@ nh3==0.2.14
 hogql-parser==1.0.21
 zxcvbn==4.4.28
 zstd==1.5.5.1
-xmlsec==1.3.14
+xmlsec==1.3.13
```

Should we add a comment here so it doesn't accidentally get upgraded again?
```suggestion
xmlsec==1.3.13 # upgrading to 1.3.14 breaks SAML
```

---

yes - good call.

I'm also going to work on tests here so CI can catch this.